src/Exception/InvalidOptionValue.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace AmpProject\Exception;
+
+use InvalidArgumentException;
+
+/**
+ * Exception thrown when an invalid option value was provided.
+ *
+ * @package ampproject/amp-toolbox
+ */
+final class InvalidOptionValue extends InvalidArgumentException implements AmpException
```

Comment:
This should be a `DomainException` instead.

Reply:
Updated.